app/src/main/res/values/strings_i18n.xml

```diff
@@ -550,6 +552,12 @@ from friends yet.</string>
   <string name="Project_goal_and_pledge" formatted="false">Project goal and pledge amounts will be shown in approximate conversions. A successfully funded project will collect your pledge in its native currency.</string>
   <string name="Project_reminders" formatted="false">Project reminders</string>
   <string name="Project_saved" formatted="false">Project saved.</string>
+  <string name="Project_status" formatted="false">Project status</string>
+  <string name="Project_status_all" formatted="false">All</string>
+  <string name="Project_status_late_pledges" formatted="false">Late pledges</string>
```

Comment:
This string needs to change from "Late pledges" to "Late pledge" (without the final "s") will require PR on web repository like this one -> https://github.com/kickstarter/kickstarter/pull/30777/files